app/utils/discogs.schemas.ts

```diff
@@ -1,16 +1,29 @@
 import z from "zod";
 
-const AlbumSchema = z.object({
+const DiscogsAlbumSchema = z.object({
   id: z.number(),
   cover_image: z.string(),
   title: z.string(),
   year: z.string().optional(),
 });
 
-export type Album = z.infer<typeof AlbumSchema>;
+const SearchAlbumSchema = z.object({
```

Comment:
```suggestion
const AlbumSchema = z.object({
```

this will be your general Album schema, it's not necessarily tied to the search API (I'd put this in a new file `collections.schemas.ts)`